pkg/controller/sparkapplication/controller.go

```diff
@@ -75,6 +75,7 @@ type Controller struct {
 	ingressURLFormat  string
 	batchSchedulerMgr *batchscheduler.SchedulerManager
 	subJobManager     submissionJobManager
+	clientSubManager  clientSubmissionPodManager
```

Comment:
clientModeSubJobManager?

Reply:
yeah that sounds better, i'll do pod instead of job